inc/saiport.h

```diff
@@ -2104,6 +2104,44 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_SUPPORTED_LINK_TRAINING_MODE,
 
+    /**
+     * @brief Enable ARS on port. This enables the control for quality measure in switch pipeline.
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_ARS_ENABLE,
+
+    /**
+     * @brief Scaling factor to account for different port speeds.
+     * Port speed is normalized using the scaling factor specified
+     * 10G:1, 25G:2.5, 40G:4, 50G:5, 100G:10, 200G:20, 400G:40
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     */
+    SAI_PORT_ATTR_ARS_PORT_LOAD_SCALING_FACTOR,
+
+    /**
+     * @brief Enable historical or past port load quality measure in switch pipeline
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_ARS_PORT_LOAD_PAST_ENABLE,
```

Comment:
again, related to specific algo chosen by the vendor. Not generic

Reply:
yes, added a valid only metacheck